### PR TITLE
Updated Readme for F-droid

### PR DESCRIPTION
**Pull request by iamflorencejay**
## PR Checklist

Please check all that apply to this PR using "x":

- [x] I have checked that this PR is not a duplicate of an existing PR (open, closed or merged)
- [x] I have checked that this PR does not introduce a breaking change
- [ ] This PR introduces breaking changes and I have provided a detailed explanation below

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting)
- [ ] Refactoring (no functional changes)
- [x] Documentation changes
- [ ] Other - Please describe:

## Fixes
Readme.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,10 @@
-# LBRY Android
+# LBRY F-Droid
```
Comment (kekkyojin) [inline README.md:1]: Thanks for this change, @LaroTayoGaming.

File: README.md [hunk 2]
```diff
@@ -1,12 +1,10 @@
...
 [![pipeline status](https://ci.lbry.tech/lbry/lbry-android/badges/master/pipeline.svg)](https://ci.lbry.tech/lbry/lbry-android/commits/master)
 [![GitHub license](https://img.shields.io/github/license/lbryio/lbry-android)](https://github.com/lbryio/lbry-android/blob/master/LICENSE)
 
 An Android browser and wallet for the [LBRY](https://lbry.com) network.
 
-
-<img src="https://spee.ch/@lbry:3f/android-08-homepage.gif" alt="LBRY Android GIF" width="384px" />
-
```
Comment (kekkyojin) [inline README.md:9]: Revert this change, rebase your commit and push to your branch again.

I see the GIF is no longer loading, but this is also affecting lbry-android so it should also be fixed on that repository and then merged here.

**Review by kekkyojin — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (iamflorencejay) [reply README.md:9]: Okay copy that

File: README.md [hunk 3]
```diff
@@ -1,13 +1,13 @@
-# LBRY Android
+# LBRY F-Droid
 [![pipeline status](https://ci.lbry.tech/lbry/lbry-android/badges/master/pipeline.svg)](https://ci.lbry.tech/lbry/lbry-android/commits/master)
 [![GitHub license](https://img.shields.io/github/license/lbryio/lbry-android)](https://github.com/lbryio/lbry-android/blob/master/LICENSE)
 
 An Android browser and wallet for the [LBRY](https://lbry.com) network.
 
+<a href="https://f-droid.org/packages/io.lbry.browser"><img src="https://raw.githubusercontent.com/f-droid/artwork/master/badge/get-it-on-en-us.png" title="Get LBRY on F-droid"></a>
```
Comment (kekkyojin) [inline README.md:7]: Remove this change.

This is the source code of the app. Final users can read the instructions on the Installation section.

Also, lbry-android doesn't provide something like this on their Readme file.

File: README.md [hunk 4]
```diff
@@ -1,13 +1,13 @@
...
 
 <img src="https://spee.ch/@lbry:3f/android-08-homepage.gif" alt="LBRY Android GIF" width="384px" />
 
-
```
Comment (kekkyojin) [inline README.md:10]: Revert this.

**Review by kekkyojin — CHANGES_REQUESTED** [review]
Perform these changes, then please squash your commits so there is only one of them to be merged here and then force push to your branch.
